app/renderer/js/pages/preference/general-section.js

```diff
@@ -120,6 +132,22 @@ class GeneralSection extends BaseComponent {
 		});
 	}
 
+	initSidebarToggle() {
+		this.$sidebarOptionSettings = document.querySelector('#ui-option-settings #sidebar-option .setting-control');
+		this.$sidebarOptionSettings.innerHTML = '';
+
+		const sidebarOption = ConfigUtil.getConfigItem('show-sidebar', false);
```

Comment:
@geeeeeeeeek something is wrong. No matter what I set here it's getting `null` initially.